src/math.py

```diff
@@ -3,4 +3,24 @@ def addition(value1, value2):
         if not isinstance(value1, int) and not isinstance(value2, int):
```

Comment:
or operator more relevant here.

Reply:
I've copied this when I was exploring circle ci. I'm removing this "addition" function completely